### PR TITLE
[TIMOB-25385] TableViewRow with Label breaks layout

### PR DESCRIPTION
**Pull request by infosia**
[TIMOB-25385](https://jira.appcelerator.org/browse/TIMOB-25385)

The first row in `TableView` is not displayed when using with `Label`. I'm pushing this to `6_3_X` too because it's critical.

```js
var win = Ti.UI.createWindow({
    backgroundColor: 'green',
    layout: "vertical",
});

var row1 = Ti.UI.createTableViewRow();
row1.add(Ti.UI.createLabel({
    text: "THIS IS THE FIRST ROW",
}));

var row2 = Ti.UI.createTableViewRow();
row2.add(Ti.UI.createLabel({
    text: "THIS IS THE SECOND ROW",
}));

var table = Ti.UI.createTableView({
    data: [row1, row2],
});

var label = Ti.UI.createLabel({
    text: "This label breaks first row",
});

win.add(label);
win.add(table);

win.open();
```

Note: This is only a workaround because I'm still not sure why table view row returns incorrect row size only for the first row.

### CODE REVIEW

File: Source/UI/src/Label.cpp [hunk 1]
```diff
@@ -66,11 +66,20 @@ namespace TitaniumWindows
...
+							const auto parent = get_parent();
+							if (parent && parent->get_apiName() == "Ti.UI.TableViewRow") {
+								skipResizing = true;
+							}
 							const auto layout = getViewLayoutDelegate<WindowsViewLayoutDelegate>();
 							const auto width  = panel->ActualWidth;
 							const auto height = panel->ActualHeight;
 							if (width > 0) {
-								label__->MaxWidth = width;
+								if (!skipResizing) label__->MaxWidth = width;
```
Comment (garymathews) [inline Source/UI/src/Label.cpp]: I know its only a workaround, but I think we always encapsulate `if` statements?
```C++
if (!skipResizing) {
    label__->MaxWidth = width;
}
```

Comment (infosia) [reply Source/UI/src/Label.cpp]: Yes. I was lazy.

**Comment by ewanharris** [comment]
@garymathews I assume other than the if statement style the CR was good? Could you merge this and #1143 if so. Thanks!

**Comment by infosia** [comment]
Should be ok to merge now.  Merging.

**Pull request merged**